db-models/comments.sql

```diff
@@ -0,0 +1,9 @@
+-- db-models/comments.sql
+
+CREATE TABLE comments (
+    comment_id SERIAL PRIMARY KEY,          -- Unique for each comment
+    parent_comment_id INTEGER,              -- ID of the parent comment for replies
```

Comment:
We'll probably want to add a foreign key relation to this as well to the comments table, though maybe unnecessary

https://www.geeksforgeeks.org/sqlite-foreign-key/